p2p/transport/webrtc/test_constants.go

```diff
@@ -0,0 +1,40 @@
+//go:build !linux
```

Comment:
It is not a test. It is used to find the interface on which to bind to run the tests on CI. Binding to `0.0.0.0` did not work on Windows and Mac CI. On linux it is simple enough to bind to the loopback interface. There were problem with binding to the non-loopback interface on linux which is why it is in a separate file.

Reply:
Binding to 0.0.0.0 works on Windows and Mac, we do it all the time.
Running code only used for tests in `init()` (which means it gets executed in production, and even worse, on bootup!) is not a good idea.

Reply:
I documented the issue I faced at the time here:
```
// non-linux builds need to bind to a non-loopback interface
// to accept incoming connections. 0.0.0.0 does not work since
// Pion will bind to a local interface which is not loopback
// and there may not be a route from, say 192.168.0.0/16 to 0.0.0.0.
```
I cannot control the address to which pion will bind. 
The gathering code is here https://github.com/pion/ice/blob/8c1adf18f14735ea6c35e8a714392f509f84957d/gather.go#L134 and https://github.com/pion/ice/blob/8c1adf18f14735ea6c35e8a714392f509f84957d/util.go#L135 .

I have the following options:
1. Add the `_test` suffix to the constants file. This will result in an awkward name but should only be compiled and executed when testing.
2. Pass the OS as a variable during tests so I can perform setup then. (I checked and the test does not have access to GOOS env var). We could also do tag based compilation here but we would have to pass the tags to the tests.

@marten-seemann do you have a preference? I'm leaning towards the first option since it would be fewer changes.

Reply:
I'm still not sure I understand the issue, but
1. this should definitely live in a _test.go file, so it's not polluting production code
2. it should not live in an `init` function

Reply:
We could move it to a test file as a setup function.

Reply:
@marten-seemann I've moved the function to new files. It should be more in line with the review.